subsys/bluetooth/mesh/scene_srv.c

```diff
@@ -33,6 +34,16 @@ static char *scene_path(char *buf, uint16_t scene, bool vnd, uint8_t page)
 	return buf;
 }
 
+static inline void pages_count(bool vnd, uint8_t page, 
+			       struct bt_mesh_scene_srv *srv)
```

Comment:
`update_page_count` or something? There's no way to tell what the purpose of this is based on the name. Also, the model pointer is always the first argument in model functions.

Reply:
Renamed.